core/src/test/java/hudson/util/io/ZipArchiverTest.java

```diff
@@ -102,6 +103,68 @@ public void backwardsSlashesOnWindows()  {
         assertEquals("foo/bar/baz/Test.txt", zipEntryName);
     }
 
+    @Test
+    public void huge64bitFile()  {
```

Comment:
Would it make sense to disable this test by default?
It may be consuming too much space for our CI and for local builds